pandas/_libs/tslibs/period.pyx

```diff
@@ -1270,6 +1270,36 @@ cdef class _Period(object):
 
     @property
     def days_in_month(self):
+        """
+        Get the total number of days in the month that this period falls on.
+
+        Returns
+        -------
+        int
+
+        See Also
+        --------
+        Period.daysinmonth : Returns the number of days in the month.
+        DatetimeIndex.daysinmonth : Return the number of days in the month.
+        calendar.monthrange : Return a tuple containing weekday
```

Comment:
In See Also section : place of Returns use Get

Reply:
apart from this LGTM

Reply:
Sure! I will make the suggested changes. Thanks for the review.